index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compactible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./styles.css" />
+    </head>
+    <body>
+        <header class="header">
+            <h1 class="logo"><a class="logo" href="#home">PETER</a></h1>
```

Comment:
- [ ] Hi @peteronyam, great effort wrapping your logo inside an `anchor` tag as suggested by the previous reviewer, but I would kindly suggest you don't make use of an `h1` tag here as in a webpage you can only use one `h1` for proper SEO and accessibility and I notice the main title has been given the `h1`. kindly use another `hx` tag if need be like: `h2 .... h6`.

> N.B: The main title (e.g., "I'm Marianne ...") is an `h1` tag. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

Reply:
@brainconnect93  You said "if need be" or that is what you require,
The previous reviewer asked me to change that to h1 tag, initially I used h2 tag there and if you check the commits history you will see it and now you decided not to accept it that I should return it to what I used initially and now I have 1 more time to submit this for review, what if the next reviewer is not comfortable with the h2 tag over there?
I need to know because you won't be affected in anyway, please

Reply:
@peteronyam As the 3rd Code Reviewer I have approved your project. I find you passionate about it. Keep it up, man.